integr8ly/manifests
# Add launcher csv

## What
Adds a Launcher CSV in preparation for adding Launcher to the Integreatly operator, and updates RHSSO version

- [x] Add Launcher CSV
     - This is generated from the [launcher-operator](https://github.com/fabric8-launcher/launcher-operator)
- [x] Add package
- [x] Update the makefile to include `push/launcher` and add it to `push/all`

## Verification Steps
1. Push the changes above to your app registry
 - Run `make push/all`
 - **Verify**: Launcher (0.1.2), RHSSO(1.8.2) and Integreatly Operator (1.6.0) should be available in your app registry
2. Create an `OperatorSource` with your app registry
 - **Verify**: Launcher and RHSSO should become available in the Operator Hub
 - **Verify**: Launcher and RHSSO CR should be available in the developer catalog
3. Install Launcher to a specific namespace
 - **Verify**: An Install plan should be created
 - **Verify**: Launcher and RHSSO operator should be deployed successfully
